Lib/email/policy.py

```diff
@@ -4,7 +4,13 @@
 
 import re
 import sys
-from email._policybase import Policy, Compat32, compat32, _extend_docstrings
+from email._policybase import (
+    _extend_docstrings,
+    Compat32,
+    compat32,
+    Policy,
+    validate_header_name
+)
```

Comment:
can you keep a case-sentitive alphabetical order actually? (I think that's how sorted imports elsewhere do it, but check that beforehand)

Reply:
Current looks good.

Reply:
I would still prefer a case-sensitive alphabetical order. We prefer consistency and/or smaller diffs when possible (does "Current looks good." actually tells me that it's how it's done elsewheere?)